packages/authentication/package.json

```diff
@@ -48,7 +48,7 @@
     "eslint": "eslint . --ext ts",
     "prettier": "prettier \"**/*.{css,html,ts,js,java}\"",
     "swiftlint": "node-swiftlint",
-    "docgen": "docgen --api FirebaseAuthenticationPlugin --output-readme README.md --output-json dist/docs.json",
+    "docgen": "docgen --api FirebaseAuthenticationPlugin --package @capacitor-firebase --output-readme README.md --output-json dist/docs.json",
```

Comment:
Does this param already work?

Reply:
No, it hasn't been merged yet. I actually forgot to remove it again...